rust/basic_bitcoin/src/basic_bitcoin/src/bitcoin_api.rs

```diff
@@ -0,0 +1,91 @@
+use ic_btc_types::{
+    GetBalanceRequest, GetCurrentFeePercentilesRequest, GetUtxosRequest, GetUtxosResponse,
+    MillisatoshiPerByte, Network, SendTransactionRequest,
+};
+use ic_cdk::{api::call::call_with_payment, export::Principal};
+
+// The fees for the various bitcoin endpoints.
+const GET_BALANCE_COST_CYCLES: u64 = 100_000_000;
+const GET_UTXOS_COST_CYCLES: u64 = 100_000_000;
+const GET_CURRENT_FEE_PERCENTILES_FEE: u64 = 100_000_000;
+const SEND_TRANSACTION_BASE_FEE: u64 = 5_000_000_000;
+const SEND_TRANSACTION_FEE_PER_BYTE: u64 = 20_000_000;
+
+/// Returns the balance of the given bitcoin address.
```

Comment:
Bitcoin should take an uppercase here (cf https://bitcoin.org/en/vocabulary#bitcoin)

Reply:
I appreciate the attention to detail, but let's not worry too much about casing, especially when it's an example and not a specification. Life's too short for that :)